addon/components/bs-spinner.js

```diff
@@ -0,0 +1,79 @@
+import Component from '@glimmer/component';
+import arg from '../utils/decorators/arg';
+
+/**
+ Implements a [Spinner element](https://getbootstrap.com/docs/5.2/components/spinners/).
+
+ ### Basic Usage
+
+ ```hbs
+ <BsSpinner/>
```

Comment:
With the recent changes I would like to have the accessible version shown here:

```suggestion
 <BsSpinner>Loading...</BsSpinner>
 
 The (optional) component's block will render as visually invisible but accessible text for assistive technology like screen readers.
```